bedita-app/models/behaviors/captions.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/*-----8<--------------------------------------------------------------------
+ * 
+ * BEdita - a semantic content management framework
+ * 
+ * Copyright 2018 ChannelWeb Srl, Chialab Srl
+ * 
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published 
+ * by the Free Software Foundation, either version 3 of the License, or 
+ * (at your option) any later version.
+ * BEdita is distributed WITHOUT ANY WARRANTY; without even the implied 
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License 
+ * version 3 along with BEdita (see LICENSE.LGPL).
+ * If not, see <http://gnu.org/licenses/lgpl-3.0.html>.
+ * 
+ *------------------------------------------------------------------->8-----
+ */
+
+/**
+ * Compact find result
+ * 
+ * in model var actAs = array("CompactResult" => array('model to esclude'));
+ *
+ */
+class CaptionsBehavior extends ModelBehavior {
+
+    var $config = array();
```

Comment:
[minor] `var` is the PHP 4 way to declare class members. Better:

```php
public $config = array();
```

(Or `protected` since `$config` is unlikely to be accessed outside of this class… and might probably be removed at all.)